lib/sequenceserver/blast/report.rb

```diff
@@ -85,9 +85,10 @@ def extract_program_info(ir)
       # matrix, evalue, gapopen, gapextend, and filters are available from XML
       # output.
       def extract_params(ir)
-        @params = Hash[
+        params = Hash[
           *ir[7].first.map { |k, v| [k.gsub('Parameters_', ''), v] }.flatten
         ].merge(job.advanced_params)
+        @params = Hash['e value', params.delete('expect')].merge!(params)
```

Comment:
This will ensure that 'e value' is at the front of the hash, so it will be the first column in the table at the top of the report